awx/settings/defaults.py

```diff
@@ -470,11 +472,21 @@ def IS_TESTING(argv=None):
     'task_manager': {
         'task': 'awx.main.scheduler.tasks.run_task_manager',
         'schedule': timedelta(seconds=20),
-        'options': {'expires': 20,}
+        'options': {'expires': 20}
+    },
+    'isolated_heartbeat': {
```

Comment:
Note that this task will now always run. The dynamic celery queue adding logic will add routes to ensure this task get serviced by controller nodes.